docs/pages/how_to_guides/how_add_feature.md

```diff
@@ -52,13 +54,13 @@ template <typename IDType, typename NNZType, typename ValueType, typename Featur
 class Feature : FeaturePreprocessType<IDType, NNZType, ValueType, FeatureType> {
 	// ...
 	Feature(float alpha, float beta){
-		this->params_ = shared_ptr<FeatureParams>(new FeatureParams{alpha, beta});
+		this->params_ = std::make_shared<FeatureParams>(new FeatureParams{alpha, beta});
```

Comment:
As parameters to make shared you only need the parameters to the constructor of `FeatureParams`, so:
`std::make_shared<FeatureParams>(alpha, beta);`